releasenotes/notes/disable-fs-group-injection.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+releaseNotes:
+- |
+  **Updated** default value of the feature flag `ENABLE_LEGACY_FSGROUP_INJECTION` to false
```

Comment:
```suggestion
  **Updated** default value of the feature flag `ENABLE_LEGACY_FSGROUP_INJECTION` to false; this may cause issues with sidecars when installing on Helm on Kubernetes versions prior to 1.19.
```

Reply:
Done